### PR TITLE
Add build-and-deploy details for theme jar

### PR DESCRIPTION
**Pull request by bickelj**
Add specific details regarding deployment of the theme jar so that anyone on the team would be able to deploy with more confidence and less head-scratching.

### CODE REVIEW

File: pdc-keycloak-theme/README.md [hunk 1]
```diff
@@ -14,6 +14,37 @@ Start in the `pdc-keycloak-theme` directory
 
 The resulting jar should be in `build/libs`. This theme jar is what should be included in keycloak's `/providers` directory.
 
+If deploying this jar, you should rename it based on the git commit. Example:
+- `cp build/libs/pdc-keycloak-theme.jar pdc-keycloak-theme-20240313-05793e4.jar`
+
+The version id here is the UTC date of the commit followed by the first seven digits of the commit SHA1 sum.
```
Comment (kfogel) [inline pdc-keycloak-theme/README.md:20]: I think that's not a "UTC" date but rather an "ISO-8601" date?

IIUC, UTC is about time zones and time of day, not about identifying a particular day.  Philosophically they are different things too: UTC is about an actual time, whereas ISO-8601 is about the *representation* of a date.

**Review by hminsky2002 — APPROVED** [review]
everything looks functional, grammatically correct and well documented

File: pdc-keycloak-theme/README.md (hunk 1 above)
Comment (bickelj) [reply pdc-keycloak-theme/README.md:20]: No and yes. "No" it is (not not) a UTC date and "yes" it is also formatted in a concise ISO 8601 format.

In other words, yes, UTC is about zones. The date chosen for the version of the jar should be the UTC zone to be unambiguous and universal. Suppose someone made a commit on the east coast at 10 pm. Well, that would be a different day in UTC.

To do: clarify the format in addition to the zone in the README!

Comment (bickelj) [reply pdc-keycloak-theme/README.md]: I updated it. And you can imagine the story behind specifying the UTC zone!

**Pull request merged**